scripts/run_doxygen.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
:no_entry_sign: Please add instructions to `doc/architecural/compilation-and-development.md`

Reply:
Done.